trussworks/react-uswds
# build: improve DangerJS checks

# Summary

Improvements to the DangerJS checks

## How To Test

Once trussworks/react-uswds#2330 is merged you can trigger the build checks workflow to see danger execute on this branch.
